cli/app.go

```diff
@@ -53,3 +64,54 @@ func (a *AMP) Status() {
 		fmt.Println("Status")
 	}
 }
+
+// Logs fetches the logs
+func (a *AMP) Logs(cmd *cobra.Command) error {
```

Comment:
Sure. Can it be done in another pull request though ? I'd like to avoid this PR to linger for a week :)